.github/workflows/luarocks.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Push to Luarocks
+
+on:
+  push:
+    tags:
+      - '*'
+  release:
+    types:
+      - created
+    tags:
+      - '*'
```

Comment:
:thinking: you can probably remove either the tag or the release trigger.

It looks like it gets triggered twice.